data-collection/src/main/java/refactoringml/RefactoringAnalyzer.java

```diff
@@ -343,7 +340,7 @@ private Yes calculateCkMetrics(String refactoredClass, String refactorCommit, Ca
 					refactorCommit,
 					refactoringDate,
 					parentCommit,
-					ck.getFile().replace(tempDir, ""),
+					FilePathUtils.enforceUnixPaths(ck.getFile()).replace(tempDir, ""),
```

Comment:
@Dahny maybe that's where Jan started to implement the cross-platform compatibility

Reply:
Yes